Dockerfiles/README.md

```diff
@@ -0,0 +1,227 @@
+# How to Use horde-worker-reGen Dockerfiles (CUDA and ROCm)
+
+This guide explains how to use the Dockerfiles for building the horde-worker-reGen application, supporting both NVIDIA (CUDA) and AMD (ROCm) GPUs.
+
+## Prerequisites
+
+- Docker installed on your system
+- Git installed on your system
+- NVIDIA GPU with appropriate drivers (for CUDA version)
+- AMD GPU with appropriate drivers (for ROCm version)
+
+## Checkout only this directory
+
+```bash
+git clone --sparse https://github.com/Haidra-Org/horde-worker-reGen.git
+cd horde-worker-reGen
+git sparse-checkout set --no-cone Dockerfiles /bridgeData_template.yaml
+```
+
+# Basic setup
+## Using docker compose
+
+If your system is set up properly (see [Prerequisites](#prerequisites))
+you can just [setup](https://github.com/Haidra-Org/horde-worker-reGen?tab=readme-ov-file#configure) your bridgeData.yaml file and then run
+```bash
+docker compose -f Dockerfiles/compse.[cuda|rocm].yaml build --pull
```

Comment:
Typo here, it should be `compose.[` instead of `compse.[`

Reply:
Good catch